### ISSUE TITLE
Improve usability of process logging page

### ISSUE THREAD

**Issue by fosterlynn** [valnet/valuenetwork#410]
From email discussion, here is Sensorica input so far:

1. There are different workflows in different networks, and on different projects and processes within one network.  For example, the "Take Task" button is for networks where collective assignment of tasks is not done, rather people self assign in an open manner.  Often people even in OVNs do not work that way.  Assigning people in the description is a sign of that. There is no reason for people to take a task and log work they have already done at one time.  Taking a task is a coordination thing, and should be done before doing the work so others are informed of the worker's intention.  If that is not needed, no need to do it.

Suggested buttons (exact wording to be worked out):
* I'll start working on this now
* I already did this
* I'll work on this later

Question: Do we want to allow assignment of tasks by others, this is what happens in practice a lot?  Doesn't mean there is hierarchy necessarily, just means there was a collective decision and the planner records all of that.

2. The page has lots of information when a lot of people are working on a longer process, and it is hard to find what you are working on.
- Descriptions are long and not well delineated
- Ordering of work requirements might be better by type of work

Some thought needs to be put into how to visually clarify the page, and possibly some filtering.

**Comment by bhaugen** [comment]
It's also way too slow, which will require eliminating all the Django forms...
